### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by ryano646**
This is the new PR to add UI commits to

### CODE REVIEW

**Comment by ryano646** [comment]
I will be adding screenshots soon

**Comment by ryano646** [comment]
### Screenshots!

collection_view
confirmation_page
default_home
different_user_view
form_page
home_no_links
home_page
link_options
login_page
max_display
max_display_collections_list
registration
user_options_dropdown

File: links/templates/links/collection_detail.html [hunk 1]
```diff
@@ -2,38 +2,71 @@
...
+                        {% csrf_token %}
+                        <input class="option" type="submit" name="button" value="Enable">
+                        <input type="hidden" name="d_id" value={{link.id}}>
+                        <input type="hidden" name="set" value="Set">
+                        <input type="hidden" name="type" value="Enable">
+                    </form>
+                    {% endif %}
+                    <label class="option" for="toggle_options{{forloop.counter}}">{% trans "Close Window" %}</label>
+                </div>
+            </div>
```
Comment (lmd414) [inline links/templates/links/collection_detail.html]: On the collection detail page, users should not be able to change links to default or enable/disable them.

**Comment by mcPython04** [comment]
Screenshots look good!

File: links/templates/links/collection_detail.html (hunk 1 above)
Comment (ryano646) [reply links/templates/links/collection_detail.html]: Oh, my bad, I guess I copy/pasted and forgot to remove that. I'll take it out.

Comment (ryano646) [reply links/templates/links/collection_detail.html]: Wait, I figured it out. That stuff was removed on the main branch, and this file doesn't have the updated changes. I don't why it didn't update when I merged the branches earlier. I'll still go ahead and fix it.

**Pull request closed without merging**